### PR TITLE
Remove ok check

### PR DESCRIPTION
**Pull request by allanchau**
This PR removed the `!ok` logic from the fetch library.

**Setup**

- [x] Include the latest beta tag into one of your projects.

**Testing**

- [x] Create an api that returns a 404 with a response body.
- [x] Make sure the response body is available in the `result` property.

### CODE REVIEW

File: packages/fetch/tests/index.test.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ it('returns a tuple for successful requests', async () => {
```
Comment (smebberson) [inline packages/fetch/tests/index.test.js:22]: @allanchau, could you please add an additional test here to make sure the Response object properties are being passed through properly.

**Review by smebberson — CHANGES_REQUESTED** [review]
@allanchau, looks good, I'd just like another test.

File: packages/fetch/tests/index.test.js (hunk 1 above)
Comment (allanchau) [reply packages/fetch/tests/index.test.js:22]: @smebberson I've added the test in now. The main properties are present, but some of the ones listed here are not. https://developer.mozilla.org/en-US/docs/Web/API/Response#properties

Comment (smebberson) [reply packages/fetch/tests/index.test.js:22]: @allanchau, yeah, no worries.

File: docusaurus/docs/fetch.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+## Usage
+
+To use `@idearium/fetch`, simply require it at the top of your file.
+
+It is simply a fetch wrapper that automatically parses the body response into a property `result`.
+
+```js
+const { ...response, result } = await fetch('...', { fetchOpts });
```
Comment (smebberson) [inline docusaurus/docs/fetch.md:29]: That isn't how you use it is it? Shouldn't it be more like...

```suggestion
const { ok, status, result } = await fetch('...', { fetchOpts });
```

**Review by smebberson — CHANGES_REQUESTED** [review]
@allanchau, I think we need a minor tweak with the usage instructions.

File: docusaurus/docs/fetch.md (hunk 2 above)
Comment (allanchau) [reply docusaurus/docs/fetch.md:29]: @smebberson Yeah, I was trying to highlight `result` is a custom property we add and the rest are just fetch response properties.

**Review by smebberson — APPROVED** [review]

**Comment by smebberson** [comment]
@allanchau, all looks good. Please make a release.

**Pull request merged**